packages/core/src/App/Containers/Modals/mt5-notification/mt5-notification-description.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+
+const MT5NotificationDescription = () => {
+    return (
+        <div className={isMobile() ? 'mt5-notification-modal-mobile' : ''}>
```

Comment:
```suggestion
        <div className={isMobile() ? 'mt5-notification-modal-mobile' : ''}>
```
there're no styles for desktop?